src/encoded/types/submission_center.py

```diff
@@ -17,3 +17,12 @@ class SubmissionCenter(Item):
     item_type = 'submission_center'
     schema = load_schema('encoded:schemas/submission_center.json')
     embedded_list = []
+
+    def __ac_local_roles__(self):
+        """This creates roles that the submission center item needs so it can be edited & viewed"""
+        roles = {}
+        sc_submitters = 'submits_for.%s' % self.uuid
+        roles[sc_submitters] = SUBMISSION_CENTER_SUBMITTER
+        sc_member = 'submission_centers.%s' % self.uuid
```

Comment:
Just minor Python question: curious why using this % style string formatting rather than f-string?